stripe.go

```diff
@@ -240,7 +240,7 @@ func (s *BackendConfiguration) CallRaw(method, path, key string, form *form.Valu
 	var body io.Reader
 	if form != nil && !form.Empty() {
 		data := form.Encode()
-		if strings.ToUpper(method) == "GET" {
+		if method == http.MethodGet {
```

Comment:
I stripped the `ToUpper` here — all calls now use `net/http` constants so it should be perfectly safe to just compare it directly against `http.MethodGet` now.